test/unit/spreadElement.spec.ts

```diff
@@ -64,3 +64,30 @@ test("Spread Element Iterable", () => {
         return JSONStringify(arr)`;
     expect(JSON.parse(util.transpileAndExecute(code))).toEqual([1, 2, 4, 8, 16, 32, 64, 128, 256]);
 });
+
+test.each(["", "string", "string with spaces", "string 1 2 3"])('Spread Element String "%s"', str => {
+    const code = `
+        const arr = [..."${str}"];
+        return JSONStringify(arr)`;
+    expect(JSON.parse(util.transpileAndExecute(code))).toEqual([...str]);
+});
+
+test.each([
+    "{ ...{ value: true } }",
+    "{ value: false, ...{ value: true } }",
+    "{ ...{ value: false }, value: true }",
+    "{ ...{ value: false }, value: false, ...{ value: true } }",
+])('SpreadAssignment "%s"', expression => {
+    const code = `
+        const obj = ${expression};
+        return obj.value;`;
+    expect(JSON.parse(util.transpileAndExecute(code))).toBe(true);
```

Comment:
The execution result here should already be a boolean, so there is no reason to JSON.parse here.